### PR TITLE
 #165305753 Fix error messages on login

### PR DESCRIPTION
**Pull request by IssaIan**
#### Description
Return a correct error message upon login with incorrect credentials
#### Type of change
- [x] Bug fix (nonbreaking change which fixes an issue)
- [ ] New feature (nonbreaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

#### How Has This Been Tested?
Please describe the tests that you ran to verify your changes
- [ ] End to end
- [ ] Integration

#### Checklist:
- [ ] My code follows the style guide for this project
- [ ] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

#### Pivotal Tracker
[165305753](https://www.pivotaltracker.com/n/projects/2326460/stories/165305753)

### CODE REVIEW

File: authors/apps/authentication/serializers.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def validate(self, data):
         # `authenticate` will return `None`. Raise an exception in this case.
         if user is None:
             raise serializers.ValidationError(
-                'A user with this email and password was not found.'
+                'Wrong email or password.'
```
Comment (Ogutu-Brian) [inline authors/apps/authentication/serializers.py]: Hello @IssaIan ,
After making this change, you should have also updated the corresponding tests that use the serializer unless this was not captured in the tests, in which case you should add the unit test for this case.

In addition, the response you would get after this change is:
```
{
    "errors": {
        "error": [
            "Wrong email or password."
        ]
    }
}
```
The key `error` should be descriptive, that is, it should point to the section where the error occurred e.g `credentials`. Check the example provided in the README

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]

File: authors/apps/authentication/serializers.py (hunk 1 above)
Comment (IssaIan) [reply authors/apps/authentication/serializers.py]: Made the changes

**Review by Ogutu-Brian — APPROVED** [review]
LGTM

**Review by a-braham — APPROVED** [review]
LGTM

**Pull request merged**